main.css

```diff
@@ -1,365 +1,784 @@
+  padding-top: 24px;
 }
 
-.main-content {
-  background-image: url(header.png);
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: cover;
-  background-color: #fff;
-  display: flex;
-  flex-direction: column;
-  flex-wrap: wrap;
-  align-content: flex-end;
-  height: 587.15px;
-  border-bottom-left-radius: 80px;
+.text {
+  padding: 30px;
 }
 
-.container {
-  padding: 15px;
+.langu2 {
+  display: none;
 }
 
-.main-container {
-  padding: 0 15px;
+.salutation {
+  font-size: 40px;
+  font-weight: 700;
+  width: 80%;
+  color: #172b4d;
+  line-height: 52px;
 }
 
-.menu a {
-  color: #6070ff;
-  text-decoration: none;
-  font-style: normal;
-  font-weight: bold;
-  font-size: 18px;
-  line-height: 20px;
+.lets-connect {
+  font-size: 16px;
+  font-weight: 500;
+  color: #7f8cff;
+  margin-top: 10px;
 }
 
-.intro-text {
-  font-style: normal;
-  font-weight: bold;
-  width: 327px;
-  height: 150px;
-  font-size: 40px;
-  line-height: 52px;
-  margin-top: 50px;
-  display: flex;
-  align-items: center;
-  color: #172b4d;
+.bars {
+  padding-right: 22px;
 }
 
-.intro-text-2 {
-  font-weight: normal;
-  font-size: 16px;
-  line-height: 28px;
-  color: #344563;
-  margin-top: 14px;
-  margin-bottom: 58px;
+.logo {
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  padding-left: 22px;
+  line-height: 20px;
+  color: #6070ff;
+  background-color: #fff;
 }
 
-.social-icons {
+.social-media {
+  list-style-type: none;
   display: flex;
-  text-align: start;
+  gap: 20px;
+  margin-right: 20px;
+  margin-top: 20px;
+  margin-left: 0%;
+  padding-bottom: 100px;
 }
 
-.social-icons li {
-  list-style: none;
-  margin-right: 20px;
+.social-media li a {
+  cursor: pointer;
+  text-decoration: none;
 }
 
-.social-icons a {
-  font-size: 25px;
-  color: #505f79;
+.menu-icon {
+  cursor: pointer;
 }
 
-.connect {
-  position: relative;
-  left: 2px;
-  top: -50px;
+.nav-bar {
+  color: #7f8cff;
 }
 
-.icons-text {
-  font-style: normal;
-  font-weight: 500;
-  font-size: 16px;
-  line-height: 24px;
+.paragraph1 {
+  margin: 10px 0;
+  width: auto;
+  margin-left: 0;
 }
 
-.projects {
-  display: grid;
-  grid-template-columns: auto;
-  border-radius: 16px;
+.first-section {
+  background-image: url('./assets/background.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+  width: 100%;
   background-color: #fff;
-  padding: 16px;
-  border: 1px solid #dfe1e6;
-  margin: 85px 14px;
+  margin-bottom: 50px;
+  padding-top: 50px;
+  border: 1px solid white;
+  border-bottom-left-radius: 110px;
 }
 
-.project-card {
+.work-section {
+  display: -ms-grid;
+  display: grid;
   width: 100%;
+  gap: 40px;
+  place-items: center;
+  margin-bottom: 58px;
+}
+
+.project-card {
   display: flex;
-  align-items: flex-start;
   flex-direction: column;
+  background: #fff;
+  padding: 16px;
+  margin: 16px;
+  border-radius: 16px;
 }
 
-.about-me-btns {
-  display: flex;
-  padding: 0;
-  list-style: none;
```

Comment:
- [ ] You have done an excellent job with your desktop version design :muscle: but there is an issue because one of your cards it's showing a list with default styles. Please, fix this in order to match the Figma design :raised_hands:

  Your design:
  ![image](https://user-images.githubusercontent.com/81522435/144634639-2cf2da64-4a42-4050-b176-9763617a0bc6.png)